src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -4,29 +4,81 @@
 Inspired by django.contrib.auth.management.commands.createsuperuser.
 (http://bit.ly/2cTgsNV)
 """
-from __future__ import unicode_literals
+import json
+from datetime import datetime
+
+from olympia.api.models import APIKey
+from olympia.users.models import Group, GroupUser
 
 from django.contrib.auth import get_user_model
+from django.contrib.auth.management.commands.createsuperuser import (
+    Command as CreateSuperUserCommand)
 from django.core import exceptions
-from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
 from django.utils.six.moves import input
 from django.utils.text import capfirst
 
 
-class Command(BaseCommand):
+class Command(CreateSuperUserCommand):
     help = '''
 Used to create a superuser. This is similar to django's createsuperuser
 command but it doesn't support any arguments. This will prompt for a username
 and email address and that's it.
 '''
-    required_fields = ['username', 'email']
+    # TODO: Use `UserProfile.REQUIRED_FIELDS`? Not sure why `username`
+    # isn't in there...
+    required_fields = ('username', 'email')
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--allow-supercreate',
```

Comment:
Any of these alternatives is better than the current situation, IMHO. `manage.py createsuperuser --allow-supercreate` is super confusing.